modules/addDate.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function displayDateTime() {
```

Comment:
- [ ] As the previous reviewer specified, Kindly refactor your function definition to use arrow syntax 😄 

e.g. 

```
const displayDateTime = () =>  {...content}

export default displayDateTime
```